cypress-ci-reporter-config.json

```diff
@@ -1,7 +1,7 @@
 {
     "reporterEnabled": "spec, mocha-junit-reporter",
     "mochaJunitReporterReporterOptions": {
-        "mochaFile": "cypress/results/results-[hash].xml",
+        "mochaFile": "cypress/results/junit/results-[hash].xml",
```

Comment:
In order to make space for the new directory without further cluttering the `cypress` directory, I'm telling this thing to write its junit files to a new directory